src/main/java/com/google/sps/servlets/YoutubeApi.java

```diff
@@ -51,8 +60,36 @@ private static YouTube getService() throws GeneralSecurityException, IOException
       VideoListResponse response =
           getService().videos().list("snippet").setKey(DEVELOPER_KEY).setId(id).execute();
       Video video = response.getItems().get(0);
-      YoutubePost newPost = new YoutubePost(
-          video.getSnippet().getTitle(), video.getSnippet().getDescription(), video.getId());
+      /**Retrieving comments before creating new post */
+
+      YouTube.CommentThreads.List request = getService()
+                                                .commentThreads()
+                                                .list("snippet")
+                                                .setVideoId(video.getId())
+                                                .setTextFormat("plainText");
+      CommentThreadListResponse commentsResponse =
+          request.setKey(DEVELOPER_KEY).setMaxResults(15L).execute();
+      List<String> plainComments = new ArrayList<String>();
+      List<Long> commentLikesList = new ArrayList<Long>();
```

Comment:
Could you create a `Comment` model instead? This way, you wouldn't need 2 lists, which can be difficult to keep track of:

```java
List<Comment> commentList = new ArrayList();
[...]
commentList.add(new Comment(snippet.getTextDisplay(), snippet.getLikeCount()));

[...]
public static class Comment {
  public final String text;
  public final long likesCount;

  Comment(String text, long likesCount) {
    this.text = text;
    this.likesCount = likesCount;
  }
}
```